### ISSUE TITLE
Works in Editor, but "Encoding name 'IBM437' not supported" in Windows

### ISSUE THREAD

**Issue by Ragzouken** [tsubaki/UnityZip#5]
Not sure if I've done something weong, but when I make a build and try to use the zipping, I get this:
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/UnityEngineDebugBindings.gen.cpp Line: 65)

ArgumentException: Encoding name 'IBM437' not supported

Parameter name: name
  at System.Text.Encoding.GetEncoding (System.String name) [0x00000] in <filename unknown>:0 

  at Ionic.Zip.ZipFile..cctor () [0x00000] in <filename unknown>:0 
Rethrow as TypeInitializationException: An exception was thrown by the type initializer for Ionic.Zip.ZipFile

**Comment by tsubaki** [comment]
Did you build for standalone?
or other platform?

**Comment by Ragzouken** [comment]
Windows / x86 standalone

**Comment by Ragzouken** [comment]
This suggests it might be a dll problem? http://answers.unity3d.com/questions/378005/ioniczipdll-does-not-work-in-build.html

**Comment by tsubaki** [comment]
Looks like same issue. Can you check it?

**Comment by Ragzouken** [comment]
Right, sorry - initially I couldn't find those DLLs on my computer but after looking in the right drive I found them and it indeed fixed my problem :)
